content/en/docs/setup/production-environment/tools/kubeadm/dual-stack-support.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Dual-stack support with kubeadm
+feature:
+  title: Dual-stack support with kubeadm
```

Comment:
Let's hold off on advertising that as a feature until it's GA (or at least until we can guarantee in which release it will graduate to stable).